src/actions/profile-view.js

```diff
@@ -939,6 +940,22 @@ export function changeImplementationFilter(
   };
 }
 
+export function changeCallTreeSummaryStrategy(
```

Comment:
I realized after I left Friday that I forgot to write function documentation. I'll push another commit with it.

Reply:
Documentation would be great, thanks!